src/components/Marketplace/AddListingScreen.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
Can we rename this component to InventoryManagerScreen or something? Just cause it encapsulates both the AddListing and ListingView components!